### PR TITLE
[Doc] Add doc for tokenSettingPrefix

### PR DESCRIPTION
**Pull request by RobertIndie**
### Motivation

* Add doc for #12132 

### Modifications

* Add doc for `tokenSettingPrefix`

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 
  
- [x] doc 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: site2/docs/reference-configuration.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
 |brokerClientTlsCiphers| Specify the tls cipher the internal client will use to negotiate during TLS Handshake. (a comma-separated list of ciphers) e.g.  [TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256]||
 |brokerClientTlsProtocols|Specify the tls protocols the broker will use to negotiate during TLS handshake. (a comma-separated list of protocol names). e.g.  `TLSv1.3`, `TLSv1.2` ||
 |ttlDurationDefaultInSeconds|The default Time to Live (TTL) for namespaces if the TTL is not configured at namespace policies. When the value is set to `0`, TTL is disabled. By default, TTL is disabled. |0|
+|tokenSettingPrefix| Configure the prefix of the token related setting like `tokenSecretKey`, `tokenPublicKey`, `tokenAuthClaim`, `tokenPublicAlg`, `tokenAudienceClaim`, and `tokenAudience`. ||
```
Comment (Huanli-Meng) [inline site2/docs/reference-configuration.md:221]:
```suggestion
|tokenSettingPrefix| Configure the prefix of the token-related settings, such as `tokenSecretKey`, `tokenPublicKey`, `tokenAuthClaim`, `tokenPublicAlg`, `tokenAudienceClaim`, and `tokenAudience`. ||
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave a minor comment. 
BTW, one more question, will docs in previous releases need to be updated? Thanks

**Comment by Anonymitaet** [comment]
Thanks for your contribution. Does this affect only master or other versioned docs? 
If latter, could you please help update [all affected versions](https://github.com/apache/pulsar/tree/master/site2/website/versioned_docs)? Thanks

**Comment by RobertIndie** [comment]
@Huanli-Meng @Anonymitaet There is no need to update the previous release doc.

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Comment by Anonymitaet** [comment]
@RobertIndie please add docs for 2.8.1 as well, thanks

**Pull request merged**